### PR TITLE
Bug 1211019 - [TaskManager] After choosing an app, that app should be…

### PR DESCRIPTION
**Pull request by mcav**
…come frontmost in the stack.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=706c3dcb4efcd434c43d9686aba965d221f715e1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cdced58ede94694daa6a02e665301d860240828a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0225874c73145d9fceafe2beff730cb43421224e

File: apps/system/js/edge_swipe_detector.js [hunk 1]
```diff
@@ -147,9 +147,7 @@
           this.lifecycleEnabled = false;
           break;
         case 'cardviewclosed':
-          if (e.detail && e.detail.newStackPosition) {
-            this.lifecycleEnabled = true;
-          }
+          this.lifecycleEnabled = true;
```
Comment (etiennesegonzac) [inline apps/system/js/edge_swipe_detector.js]: Exiting the cardview to the homescreen will enable the edge zones with this change. We don't want that :)
But it should be fine with just checking that the next app isn't a homescreen.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ccf35ee69345c8d4bf771180c89c8754bfbff7b9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f0af272d083dd29b4ebf7697bec42d6fc8ee4702

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=008c6b292531c675e31dfda4ae3f886a00f59941

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e652686289a8b1bed1ac2173394dd0b2b95aadba

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e652686289a8b1bed1ac2173394dd0b2b95aadba

**Pull request merged**